qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -613,6 +613,34 @@ def delete_raw_data(self, study, user, callback):
 
         callback((msg, msg_level, tab, tab_id, None))
 
+    def delete_prep_template(self, study, user, callback):
+        """Delete the selected prep template
+
+        Parameters
+        ----------
+        study : Study
+            The current study object
+        user : User
+            The current user object
+        callback : function
+            The callback function to call with the results once the processing
+            is done
+        """
+        prep_template_id = int(self.get_argument('prep_template_id'))
+        tab_id = PrepTemplate(prep_template_id).raw_data
```

Comment:
It seems like `tab_id` could easily be replaced by `prep_id`. Plus I don't understand why the name of this variable is `tab..`.